### ISSUE TITLE
About the pixel-wise loss

### ISSUE THREAD

**Issue by caozhenxiang-kouji** [yiyuan1991/3D-Face-GCNs#20]
Thanks for the great work! We notice that you are using project_loss_image instead of the gcn_render_loss in the code, would you please explain the project_loss_image a little bit more detailed? Does the project_loss_image perform better than the gcn_render_loss?

**Comment by HOMGH** [comment]
> Thanks for the great work! We notice that you are using project_loss_image instead of the gcn_render_loss in the code, would you please explain the project_loss_image a little bit more detailed? Does the project_loss_image perform better than the gcn_render_loss?

Yes I noticed it as well. @FuxiCV  Apparently you are not using gcn_render_loss and pca_render_loss. Right?
